src/index.html

```diff
@@ -1,13 +1,62 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body>
+
+    <div class="card"  data-qa="card">
+
+      <img
+        src="./images/imac.jpeg"
+        alt="Apple iMac"
+        class="product-photo"
+      >
+
+      <div class="product">
+        <div class="product__info">
+          <h3 class="product__title">
+            Моноблок APPLE A1419 iMac 27&quot; Retina 5K (MNED2UA/A)
+          </h3>
+          <p>Код товара: 195434</p>
+        </div>
+
+        <div class="product__rating">
+          <div class="stars">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <div>
+            <p>Отзывов: 5</p>
+          </div>
+        </div>
+
+        <div class="product__price">
+          <p>Цена:</p>
+          <p class="price-number">69 999 грн</p>
```

Comment:
no need to make a separate BEM-block for price. Most probably, it won't be reused as an independent block. So let's make it product__price-number

Reply:
okay :)